packages/web/src/forms/FormComponents.tsx

```diff
@@ -189,13 +205,93 @@ export function LabeledInput({ name, multiline, hasError, value, onInput, label
           fonts.p,
           styles.input,
           standardStyles.elementalMarginBottom,
+          isDarkMode && standardStyles.inputDarkMode,
           hasError && styles.errorBorder,
         ]}
-        focusStyle={standardStyles.inputFocused}
+        focusStyle={isDarkMode ? standardStyles.inputDarkFocused : standardStyles.inputFocused}
         name={name}
         value={value}
         onChange={onInput}
       />
     </>
   )
 }
+
+interface CheckboxProps {
+  checked: boolean
+  name: string
+  onPress: (x: any) => void
+}
+
+export function Checkbox({ checked, onPress, name }: CheckboxProps) {
```

Comment:
Long term, it'd be better for these to each be in their own file to enable better code splitting and lazy loading.